README.md

```diff
@@ -73,10 +73,17 @@ pg_ctl start -D data
 elasticmq& # queues are preconfigured, so no additional work needed
 ```
 
+With `pg_ctl` the PostgreSQL server will keep running until you restart your system. 
+On the other hand ElasticMQ (`elasticmq`) will stop when you close the nix shell, 
+it's only needed if you want to work on asynchronous/worker-related tasks. 
```

Comment:
Is this actually the case? I run the command `elasticmq&` to start it in the background and then exit nix and haven't run into issues.

Reply:
You are right! Updated it!